presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/RemoveUnsupportedDynamicFilters.java

```diff
@@ -127,7 +127,7 @@ public PlanWithConsumedDynamicFilters visitJoin(JoinNode node, Set<String> allow
                         right,
                         node.getCriteria(),
                         node.getOutputSymbols(),
-                        node.getFilter(),
+                        node.getFilter().map(this::removeAllDynamicFilters),  // no DF support at Join operators.
```

Comment:
I think we should extend `DynamicFiltersChecker` to verify that there are no DFs at Join operator and add a test to `TestDynamicFiltersChecker`

Reply:
Good point, fixed below.